service/http-api-samples/pom.xml

```diff
@@ -12,6 +12,7 @@
     <artifactId>fido-iot</artifactId>
     <groupId>org.fido</groupId>
     <version>0.3-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
```

Comment:
This might not be needed due to the merged PR#64 https://github.com/secure-device-onboard/pri-fidoiot/pull/64. Please check.

Reply:
You are correct.  Merging a revert in the next few minutes.